### PR TITLE
Update the list of supported clientScript variables

### PR DESCRIPTION
**Pull request by mistydemeo**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -662,9 +687,6 @@ command scripts.
 +------------------------+-----------------------------+--------------------------------------------+---------------------------------------------------------+
 | %fileExtensionWithDot% |  --file-extension-with-dot= |  As above, without stripping the period.   |  .mov                                                   |
 +------------------------+-----------------------------+--------------------------------------------+---------------------------------------------------------+
-| %outputDirectory%      |  --output-directory=        |  The directory to which the output file    |  /path/to/access/copies                                 |
```
Comment (mistydemeo) [inline docs/index.rst:665]: This was accidentally documented twice - I removed the less useful of the two definitions.

**Comment by Hwesta** [comment]
:+1:

**Comment by jhsimpson** [comment]
:+1:

**Comment by sromkey** [comment]
Looks great to me :+1:

**Pull request merged**